package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
If you want to get the `package-lock.json` in order you need to have npm 7 installed. Running `npm install -g npm` should most likely help to update the npm version. If there are any problems I can regenerate the file later though :)

Reply:
This helped. Thanks @jonatanklosko.

Reply:
Actually I see this `package-lock.json` is in the root directory, probably because of running `npm install` there by accident. Please remove this one and regenerate the actual lockfile under assets :)

Reply:
Ah, I see. I am not so familiar with npm so I had no idea why it was there but good catch.

Reply:
I think you also need to update your npm version because this project is using lockfileVersion: 2.

Reply:
@jonatanklosko Just deleted it. 

@josevalim strange, currently the `package-lock.json` file in assets/ is using lockfileVersion: 2, also I am using npm version 7.16.0.

``"name": "assets",
  "lockfileVersion": 2,`
```